extensions/blocks/videopress/editor.js

```diff
@@ -22,12 +49,19 @@ const addVideoPressSupport = ( settings, name ) => {
 	}
 
 	const { attributes, deprecated, edit, save, supports, transforms } = settings;
-
 	const { available, unavailableReason } = getJetpackExtensionAvailability( 'videopress' );
 
+	// Check if VideoPress is unavailable and filter the mediaplaceholder to limit options
+	if ( [ 'missing_plan', 'unknown' ].includes( unavailableReason ) ) {
+		addFilter( 'editor.MediaPlaceholder', 'jetpack/videopress', videoPressNoPlanMediaPlaceholder );
```

Comment:
I'm not sure if this would be the right place, but we should also add the filter to include the extra CSS classes when the nudge is displayed. From what I can tell, this is so that the borders display properly and to override the `pointer-events: none` setting applied to blocks with warnings.

https://github.com/Automattic/jetpack/blob/master/extensions/shared/register-jetpack-block.js#L54

Reply:
I tried to find a way to do this within `wrapPaidPlan` but it caused a loop because every time we added the filter the component was rerendering because of the way the filter is applied.

Reply:
I pushed a commit to add this.

Reply:
@scruffian Should I add in the filter manually in that case?